/Robot_Link_Initial_Pose:
  ros__parameters:
    initial_pose: [5.0,0.0,0.0,0.0,0.0,0.0]

static_transforms:
    point_A: ['2', '0', '1', '1.57', '0', '0', 'world', 'A']
    point_B: ['2', '0', '1', '1.57', '0', '0', 'A', 'B']
    point_C: ['2', '0', '1', '1.57', '0', '0', 'B', 'C']
    point_D: ['2', '0', '1', '1.57', '0', '0', 'C', 'D']
    point_E: ['2', '0', '1', '1.57', '0', '0', 'D', 'E']
